src/Algebra/Module.agda

```diff
@@ -0,0 +1,285 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definitions of algebraic structures defined over some other
+-- structure, like modules and vector spaces
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Module where
```

Comment:
Given #876 can we rename this `Algebra.Module.Packages`? The `Packages` part of it may change before the next release, but at least we'll have it in the right place.